transport/TransportImpl.h

```diff
@@ -118,6 +118,7 @@ class TransportImpl : public RtcTransport,
     void setRemoteIce(const std::pair<std::string, std::string>& credentials,
         const ice::IceCandidates& candidates,
         memory::AudioPacketPoolAllocator& allocator) override;
+    void addRemoteIceCandidate(const ice::IceCandidate& candidate) override;
```

Comment:
This allows client side to remote candidates later and have those probed. We do not use trickling from smb side, but it is used by clients and we need to test it somehow.